templates/dart/lib/services/service.dart.twig

```diff
@@ -16,11 +16,11 @@ class {{ service.name | caseUcfirst }} extends Service {
     {{ service.name | caseUcfirst }}(Client client): super(client);
 {% for method in service.methods %}
 
-     /// {{ method.title }}
+    /// {{ method.title }}
 {% if method.description %}
-     ///
+    ///
 {{ method.description|dartComment }}
```

Comment:
dartComment is for dart style comment `///` where as I found that rust uses `//` may be we can introduce new filter?

Reply:
https://doc.rust-lang.org/beta/rust-by-example/meta/doc.html#doc-comments